docs/vectorizer-worker.md

```diff
@@ -31,49 +31,49 @@ To run vectorizer workers, you need to:
+To be able to run vectorizers in your self-hosted database, use one of the following setups:
 
-- [Install a local developer environment Docker image](#install-a-local-developer-environment-docker-image): a Docker image containing a database instance and pgai vectorizer worker 
-- [Install the vectorizer worker Docker image](#install-the-vectorizer-worker-docker-image): a Docker image you use to run vectorizers on any self-hosted Postgres database with the pgai 
-   extension activated 
-- [Install vectorizer worker in your local environment](#install-vectorizer-worker-in-your-local-environment): install pgai locally so you can run vectorizers on any self-hosted
+- [End-to-end vectorizer worker with Docker Compose](#end-to-end-vectorizer-worker-with-docker-compose): a Docker Compose configuration with a database instance and pgai vectorizer worker
+- [Standalone vectorizer worker with Docker](#standalone-vectorizer-worker-with-docker): a Docker image you use to run vectorizers on any self-hosted Postgres database with the pgai
+   extension activated
+- [Install vectorizer worker as a python package](#install-vectorizer-worker-as-a-python-package): install pgai as a python package so you can run vectorizers on any self-hosted
   Postgres database with the pgai extension activated
 
-### Install a local developer environment Docker image
+### End-to-end vectorizer worker with Docker Compose
 
-The local developer environment is a docker configuration you use to develop and test pgai, vectorizers and vectorizer 
-worker locally. It includes a: 
-- Postgres deployment image with the TimescaleDB and pgai extensions installed
-- pgai vectorizer worker image  
+The end-to-end vectorizer worker is a batteries-included Docker Compose
+configuration which you use to test pgai, vectorizers and vectorizer worker
+locally. It includes a:
+- local Postgres instance with pgai installed,
+- embedding API service (Ollama)
+- pgai vectorizer worker
 
 On your local machine:
 
-1. **Create the Docker configuration for a local developer environment**
-
-   1. Copy the following configuration into a file named `docker-compose.yml`:
-   
-       ```yaml
-       name: pgai
-       services:
-         db:
-           image: timescale/timescaledb-ha:cicd-024349a-arm64
-           environment:
-             POSTGRES_PASSWORD: postgres
-             OPENAI_API_KEY: <your-api-key>
-           ports:
-             - "5432:5432"
-           volumes:
-             - ./data:/var/lib/postgresql/data
-         vectorizer-worker:
-           image: timescale/pgai-vectorizer-worker:v0.2.1
-           environment:
-             PGAI_VECTORIZER_WORKER_DB_URL: postgres://postgres:postgres@db:5432/postgres
-             OPENAI_API_KEY: <your-api-key>
-       ```
-
-   1. Replace the instances of `OPENAI_API_KEY` with a key from your AI provider.
+1. **Copy the following configuration into a file named `docker-compose.yml`**
```

Comment:
Super nit and opinionated, but could skip it 😄 
```suggestion
1. **Copy the following configuration into a file named `docker-compose.yaml`**
```

Reply:
Done. Also TIL that they changed the name to `compose.yaml` in 2021 (https://stackoverflow.com/a/76752628). Old habits die hard.